src/main/java/nl/tudelft/contextproject/model/level/util/CorridorType.java

```diff
@@ -0,0 +1,155 @@
+package nl.tudelft.contextproject.model.level.util;
+
+import nl.tudelft.contextproject.model.level.MazeTile;
+import nl.tudelft.contextproject.model.level.TileType;
+
+/**
+ * Enum for different kinds of corridor types.
+ * Used in CorridorBeautification.
+ * <P>
+ * In this class, the java doc each enum represents a certain configuration of a corridor.
+ * Each corridor is drawn, with A being the current corridor tile and C the corridor tiles around it.
+ * The horizontal corridors and vertical corridors are just as shown, but the corner corridors are defined
+ * from the direction where the extension will take place, as it was easier to keep track of the position that way.
+ * 
+ */
+public enum CorridorType {
+
+	/**
+	 * A Vertical corridor.
+	 * <p>
+	 * &ensp;C&ensp;<br>
```

Comment:
I figured they were HTML symbols, however for some reason they weren't rendered before.